src/etl/impl/Transformer.h

```diff
@@ -238,7 +243,7 @@ class Transformer
                 LOG(log_.debug()) << "object neighbors not included. using cache";
 
                 if (!backend_->cache().isFull() || backend_->cache().latestLedgerSequence() != lgrInfo.seq - 1)
-                    throw std::runtime_error("Cache is not full, but object neighbors were not included");
+                    throw std::logic_error("Cache is not full, but object neighbors were not included");
```

Comment:
Previously this would have been caught by the try/catch that attempts to catch amendment block situation. I don't think this error here can happen if Clio is setup correctly so I'd consider this a logic error and we can either have a separate catch clause for it or let it bubble up to main (i believe that was the original behaviour).